Tests/MapboxCoreNavigationTests/PredictiveCache/PredictiveCacheManagerTests.swift

```diff
@@ -0,0 +1,93 @@
+import XCTest
+import MapboxCommon
+import MapboxNavigationNative
+@testable import TestHelper
+@testable import MapboxCoreNavigation
+
+final class PredictiveCacheManagerTests: TestCase {
+    var predictiveCacheOptions = PredictiveCacheOptions()
+    let navigator = NativeNavigatorSpy()
```

Comment:
These instances are shared between tests which compromises tests isolation. Consider recreating them in `setUp` or provide computed getters.

Reply:
I moved creation of spies to setUp